pkg/bspatch/bspatch.go

```diff
@@ -26,15 +26,23 @@
 package bspatch
 
 import (
+	"bufio"
 	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
+	"os"
 
 	"github.com/dsnet/compress/bzip2"
 	"github.com/gabstv/go-bsdiff/pkg/util"
 )
 
+// Variables for streaming
+var (
```

Comment:
- use `const`
- make these private

Reply:
I'm using var to be able to vary them locally in the testing file. Not too sure we need to test different buffer sizes but they did help me catch some errors in the implementation before the `len(p)/2` change.